src/test/java/org/folio/rest/imp/InvoicesTest.java

```diff
@@ -262,5 +349,92 @@ public static String getMockData(String path) throws IOException {
         return sb.toString();
       }
     }
+  }
+  
+	public static class MockServer {
```

Comment:
Please move `MockServer` to separate class

Reply:
Done